addons/docs/package.json

```diff
@@ -120,6 +118,8 @@
     "@babel/core": "^7.11.5",
     "@storybook/vue": "6.1.0-alpha.34",
     "babel-loader": "^8.0.0",
+    "react": "15 || 16 || ^17.0.0",
```

Comment:
This is wrong. Addon-docs requires hooks which require 16.8.

Reply:
Do you have in mind other packages requiring React 16.8+? Do you prefer I use `^16.8.3 || ^17.0.0` everywhere to be more confident about the compatibility?

Reply:
Yes I think that's safe